src/app/components/Select.jsx

```diff
@@ -92,7 +93,7 @@ class Select extends Component {
                         onBlur={this.handleFocus}
                         value={this.props.selectedOption}
                     >
-                        <option value={this.props.defaultOption}>{this.props.defaultOption || "Select an option"}</option>
+                        {this.props.showDefaultOption && <option value={this.props.defaultOption}>{this.props.defaultOption || "Select an option"}</option>}
```

Comment:
Confused by how this line works, could you explain?

Reply:
Ok, so it is react basic feature  called `Conditional rendering` : https://reactjs.org/docs/conditional-rendering.html
If the left side is true the right will be rendered if false  it will not be rendered 
https://github.com/ONSdigital/florence/pull/652/files/7de944405af4195cce5c3af5a319531753a0f74f#diff-f94d43f9b0f153fdbec7fc7c3e806288c152356c039347ecf26694a099860508R32